### PR TITLE
Update Ifbyphone to DialogTech in Partner Directory

### PR DESCRIPTION
**Pull request by clarissabee**
@julieritchie Given Ifbyphone's rebranding to DialogTech, updating their name, website, and company description here. I will send you their new logo via e-mail as well. Thanks!

### CODE REVIEW

File: website/partners/technology/ifbyphone/index.hbs [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 integrations: true
 priority: 52
-summary: "Track phone calls as an offline conversion goal using ifbyphone."
-logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/ifbyphone.png"
-website_link: "http://public.ifbyphone.com/"
-website_display: "www.ifbyphone.com"
+summary: "Track phone calls as an offline conversion goal using DialogTech (formerly Ifbyphone)."
+logo: ""
+website_link: "http://www.dialogtech.com/"
+website_display: "www.dialogtech.com/"
```
Comment (stewsmith) [inline website/partners/technology/ifbyphone/index.hbs]: Could you remove the trailing `/` here?

File: website/partners/technology/ifbyphone/index.hbs [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-title: "ifbyphone"
+title: "DialogTech"
 integrations: true
 priority: 52
-summary: "Track phone calls as an offline conversion goal using ifbyphone."
-logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/ifbyphone.png"
-website_link: "http://public.ifbyphone.com/"
-website_display: "www.ifbyphone.com"
+summary: "Track phone calls as an offline conversion goal using DialogTech (formerly Ifbyphone)."
+logo: ""
```
Comment (stewsmith) [inline website/partners/technology/ifbyphone/index.hbs]: This needs a logo. It is currently blank in [the preview](http://optimizely-marketing-site-staging.s3-website-us-east-1.amazonaws.com/cbukhan/partnerdirectory-update-ifbyphone/partners/technology/)

**Comment by stewsmith** [comment]
@julieritchie Will you let me know when this is ready to merge? I'm trying to clear out a bunch of PRs today.

**Comment by kylerush** [comment]
@julieritchie can you let us know what's going on with this PR? we don't want it to sit in our queue for much longer. thank you!

**Comment by julieritchie** [comment]
@kylerush I work on a weekly cadence with these PR requests. I've noticed Stew and DFP pinging me a lot about these but the reality is that I'm getting to these as fast as I can. Now Kelly is with us and I've asked her to look at this this week :)

**Comment by dtothefp** [comment]
@julieritchie what's the status on this?

**Comment by kellyfinn** [comment]
@julieritchie @dtothefp @stewsmith This is ready- logo added and trailing "/" removed.

File: website/partners/technology/ifbyphone/index.hbs [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-summary: "Track phone calls as an offline conversion goal using ifbyphone."
-logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/ifbyphone.png"
-website_link: "http://public.ifbyphone.com/"
-website_display: "www.ifbyphone.com"
+summary: "Track phone calls as an offline conversion goal using DialogTech (formerly Ifbyphone)."
+logo: "https://s3.amazonaws.com/cloud_cannon_production/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/technology/dialogtech.png?timestamp=e6df0ff141fa0be953f0d27d7c70f51d"
+website_link: "http://www.dialogtech.com/"
+website_display: "www.dialogtech.com"
 kb_article: "https://help.optimizely.com/hc/en-us/articles/202984310"
 stars: 0
```
Comment (dtothefp) [inline website/partners/technology/ifbyphone/index.hbs]: @kellyfinn please remember to remove `https:` protocols

**Comment by dtothefp** [comment]
@kellyfinn @clarissabee changed name of directory to match new title name. This makes the url path on the website match the partner name

**Pull request merged**